Kickstarter-iOS/Tests/Views/FundingGraphViewTests.swift

```diff
@@ -92,6 +92,34 @@ internal final class FundingGraphViewTests: TestCase {
       FBSnapshotVerifyView(graphView, identifier: "state_\(key)")
     }
   }
+
+  func testOneDayProject() {
```

Comment:
added a new screenshot test for weird, edge-case one day projects cause i started getting worried i was gonna mess something up with the funding graph.

/cc @luoser